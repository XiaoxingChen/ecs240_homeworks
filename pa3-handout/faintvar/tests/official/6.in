c Branches (large)
p 20 100 278 1 100
b 1 
e 1 2
e 1 35
e 1 4
e 1 5
e 1 6
e 1 7
e 1 42
e 1 12
e 1 13
e 1 15
e 1 18
e 1 50
e 1 23
b 2 12 10 8 12 4 6 11 17 3 8
e 2 3
e 2 4
e 2 37
e 2 8
e 2 9
e 2 11
e 2 17
e 2 24
e 2 59
b 3 13 18
e 3 65
e 3 66
e 3 36
e 3 58
e 3 6
e 3 8
e 3 72
e 3 10
e 3 11
e 3 13
e 3 79
e 3 22
e 3 57
e 3 26
e 3 27
b 4 
e 4 7
e 4 9
e 4 11
e 4 47
e 4 51
e 4 21
e 4 22
e 4 54
b 5 
e 5 68
e 5 6
e 5 7
e 5 9
e 5 10
e 5 13
e 5 14
e 5 47
e 5 86
e 5 28
e 5 63
b 6 13 12 6 6 10 1 14 1 20 5 5 3 4 4 17
e 6 97
e 6 38
e 6 7
e 6 8
e 6 18
b 7 
e 7 20
e 7 60
b 8 17 17 4 3 15 5 19
e 8 44
e 8 13
e 8 46
e 8 21
e 8 26
e 8 27
b 9 2 12 19 5 9 6 5 7 5 20 10
e 9 16
e 9 58
e 9 82
b 10 17 6 4 6 4 20 12
e 10 16
e 10 70
e 10 12
e 10 46
b 11 11 1 14 1 16 7 13 11 18 10 19 10 14 20 15 14 16 2 10 15
e 11 34
e 11 35
e 11 41
e 11 62
e 11 63
b 12 3 20 18 8 5 6 16 14 15 5 3 2 10 1 8 13
e 12 32
e 12 75
e 12 71
b 13 
e 13 21
e 13 22
e 13 24
e 13 29
e 13 31
b 14 1 6 5 16 11 12 7 20 7 11 6 12 10 10 16 16 7
e 14 25
e 14 18
e 14 29
e 14 46
b 15 
e 15 74
e 15 43
e 15 44
e 15 47
e 15 48
e 15 17
e 15 80
b 16 16 14 8 18 6 4 7 4 14 18 10 10 14 17 12 7 7
e 16 24
e 16 71
b 17 
e 17 45
e 17 46
e 17 19
e 17 20
e 17 21
e 17 26
e 17 28
b 18 14 10 5 1 5 20 7 4 15 17 19 8 14 14 10 11 4 10 20
e 18 97
e 18 27
b 19 4 17 8 3 17 1 18 14 17 14 15 6 2
e 19 79
e 19 24
e 19 57
e 19 56
e 19 30
b 20 
e 20 34
e 20 40
e 20 75
e 20 45
e 20 54
e 20 88
e 20 92
b 21 7 20 19 11 20 11 2 16 5 9 6 7
e 21 25
e 21 29
b 22 
e 22 81
e 22 57
e 22 26
e 22 37
b 23 12 3
e 23 25
e 23 58
b 24 15 17 14 14 3 6
e 24 33
e 24 37
e 24 71
e 24 42
e 24 45
e 24 92
e 24 30
b 25 
e 25 66
e 25 44
b 26 
e 26 100
b 27 
e 27 40
e 27 50
e 27 77
e 27 29
b 28 
e 28 34
e 28 45
e 28 78
b 29 
e 29 72
e 29 41
e 29 48
e 29 86
e 29 30
b 30 
e 30 72
e 30 43
e 30 76
e 30 84
b 31 11 18 9 17
e 31 34
b 32 
e 32 64
e 32 67
e 32 39
e 32 40
e 32 49
e 32 86
b 33 20 4 14 3 6 15 3 16 11 1 18 19 3 17 17 19 20
e 33 35
e 33 41
e 33 42
e 33 50
e 33 84
b 34 
e 34 51
e 34 39
b 35 
e 35 100
b 36 8 4 9 5 9 12 6 4 18 11 9 15 12
e 36 40
e 36 58
e 36 69
b 37 10 2 18 12 12 2 14 14 11 13 3 7 14
e 37 100
b 38 
e 38 93
b 39 2 1 7 1 10 1 17 13 16 4 15 10 16 7
e 39 90
e 39 61
b 40 
e 40 100
b 41 2 9 15 19 6 13 8 4 8 4 6 9 11 8 11 8
e 41 73
e 41 42
e 41 79
e 41 48
e 41 55
b 42 
e 42 51
b 43 
e 43 93
e 43 53
e 43 94
b 44 
e 44 92
e 44 78
b 45 
e 45 89
e 45 51
e 45 79
b 46 
e 46 64
e 46 75
e 46 59
e 46 96
b 47 
e 47 49
e 47 50
e 47 52
e 47 68
b 48 
e 48 74
e 48 83
e 48 77
b 49 
e 49 84
e 49 70
e 49 95
b 50 7 12 9 13 2 1 9 12 10 2 4 20 9 12 4 11 9 2 6
e 50 78
b 51 
e 51 98
e 51 54
b 52 3 9 4 3 6 5 11 9 10 20 3 15 17 6 15 1 13 12
e 52 100
b 53 
e 53 96
e 53 77
b 54 
e 54 65
e 54 91
e 54 67
e 54 94
b 55 
e 55 57
e 55 61
b 56 2 9 1 18 4
e 56 98
e 56 70
b 57 1 3 20 11 18 20 15 17
e 57 81
e 57 97
e 57 62
e 57 63
b 58 
e 58 80
b 59 
e 59 100
b 60 
e 60 90
b 61 
e 61 85
b 62 
e 62 65
e 62 90
b 63 17 18 11 13 6 8 18
e 63 81
e 63 92
e 63 97
e 63 87
b 64 
e 64 100
b 65 
e 65 100
b 66 
e 66 83
b 67 
e 67 85
e 67 70
b 68 
e 68 80
b 69 
e 69 100
b 70 
e 70 87
e 70 71
b 71 
e 71 94
e 71 99
e 71 78
b 72 
e 72 96
e 72 95
b 73 
e 73 91
b 74 0 14 9 2 8 19 12 12 4 18 1 4 8 1 6 17 14 4 13 3
e 74 83
b 75 4 15 17 20 18 7
e 75 100
b 76 16 4 2
e 76 98
b 77 
e 77 84
b 78 
e 78 88
b 79 
e 79 100
b 80 
e 80 98
b 81 
e 81 100
b 82 9 3 2 13 2 4 16 5 13 7 6 10
e 82 100
b 83 
e 83 100
b 84 15 5 12
e 84 91
b 85 
e 85 88
b 86 18 1
e 86 91
e 86 94
b 87 0 14 7 3 9 17 5 7
e 87 100
b 88 
e 88 100
b 89 4 18 17 3 17 19 1 19 19 17 8 15 10 15 18 1 18 8 8 13 1
e 89 100
b 90 
e 90 100
b 91 
e 91 100
b 92 19
e 92 96
b 93 
e 93 100
b 94 7 10 14 12 6 3 17 3 19 4 10 10 7 13 7
e 94 100
b 95 
e 95 100
b 96 
e 96 100
b 97 19 15 13 9 13 4 9 19 6 7 1 12 7 4 9 8 4
e 97 100
b 98 20 11 20 2 1 5
e 98 100
b 99 
e 99 100
b 100 