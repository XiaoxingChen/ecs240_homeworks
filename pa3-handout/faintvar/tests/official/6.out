fvin 1
fvin 2
fvin 3 13
fvin 4
fvin 5
fvin 6 13
fvin 7 1 2 4 6 8 10 11 12 13 15 16 18 19 20
fvin 8 10
fvin 9 2 10 15 16 20
fvin 10 17
fvin 11
fvin 12 4 11 12 19
fvin 13 10
fvin 14 1
fvin 15 10
fvin 16 2 10 15 16 20
fvin 17 10
fvin 18 16
fvin 19 4 10 16 20
fvin 20 1 2 4 6 8 10 11 12 13 15 16 18 19 20
fvin 21 10
fvin 22 1 2 4 10 12 15 16 17 19 20
fvin 23 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 24 2 10 15 16 20
fvin 25 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 26 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 27 10 16 20
fvin 28 1 2 4 6 8 10 11 12 13 15 16 18 19 20
fvin 29 10 16 20
fvin 30 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 31 1 2 4 6 8 10 11 12 13 15 16 18 19 20
fvin 32 1 2 4 6 8 10 11 12 13 15 16 18 19 20
fvin 33 2 10 16 20
fvin 34 1 2 4 6 8 10 11 12 13 15 16 18 19 20
fvin 35 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 36 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 37 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 38 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 39 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 40 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 41 2 10 16 20
fvin 42 1 2 4 6 8 10 11 12 13 15 16 18 19 20
fvin 43 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 44 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 45 1 2 4 6 8 10 11 12 13 15 16 18 19 20
fvin 46 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 47 1 2 4 6 8 10 11 12 13 15 16 18 19 20
fvin 48 5 7 10 11 15 16 20
fvin 49 1 2 4 6 8 10 11 12 13 15 16 18 19 20
fvin 50 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 51 1 2 4 6 8 10 11 12 13 15 16 18 19 20
fvin 52 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 53 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 54 1 2 4 6 8 10 11 12 13 15 16 18 19 20
fvin 55 1 2 4 10 12 15 16 17 19 20
fvin 56 1 2 4 6 8 10 11 12 13 15 16 18 19 20
fvin 57 1 2 4 10 12 15 16 17 19 20
fvin 58 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 59 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 60 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 61 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 62 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 63 1 2 4 10 12 15 16 17 19 20
fvin 64 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 65 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 66 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 67 1 2 4 6 8 10 11 12 13 15 16 18 19 20
fvin 68 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 69 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 70 1 2 4 6 8 10 11 12 13 15 16 18 19 20
fvin 71 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 72 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 73 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 74 5 7 10 11 15 16 20
fvin 75 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 76 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 77 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 78 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 79 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 80 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 81 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 82 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 83 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 84 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 85 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 86 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 87 1 2 4 6 8 10 11 12 13 15 16 18 19 20
fvin 88 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 89 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 90 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 91 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 92 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 93 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 94 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 95 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 96 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 97 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 98 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 99 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
fvin 100 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20
